client/packages/invoices/src/Prescriptions/api/operations.graphql

```diff
@@ -8,12 +8,118 @@ fragment PrescriptionRow on InvoiceNode {
   invoiceNumber
   otherPartyId
   otherPartyName
+  clinicianId
   type
   status
   colour
   pricing {
     __typename
     totalAfterTax
+    totalBeforeTax
+    stockTotalBeforeTax
+    stockTotalAfterTax
+    serviceTotalAfterTax
+    serviceTotalBeforeTax
+    taxPercentage
+  }
+
+  user {
+    __typename
+    username
+    email
+  }
+
+  lines {
+    __typename
+    nodes {
+      ...PrescriptionLine
+    }
+    totalCount
+  }
+
+  patient {
+    __typename
+    id
+    name
+    code
+    isDeceased
+    programEnrolments {
```

Comment:
are the program enrolments required for the prescription?

Reply:
The Patient Fragment that is being used for the PatientSearchInput has it, so if we remove it, the types don't match